spec/mailers/anonymized_data_download_mailer_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+include ActiveJob::TestHelper
+
+RSpec.describe AnonymizedDataDownloadMailer, type: :mailer do
+  let (:sender_email) { 'help@simple.org' }
+  let (:recipient_name) { 'Example User' }
+  let (:recipient_email) { 'user@example.com' }
+  let (:attachment_file_names) { %w[patients.csv blood_pressures.csv medicines.csv appointments.csv sms_reminders.csv phone_calls.csv] }
+  let (:empty_attachment_data) { Hash.new }
+  let (:attachment_data) {
```

Comment:
This can potentially be moved to a fixture to clean up the spec file.

Reply:
Done. Moved attachment data to a JSON fixture.